migrator/migrations/m_172_to_m_173_network_flows_partition/stores/updated/store.go

```diff
@@ -170,8 +163,10 @@ func (s *flowStoreImpl) readRows(rows pgx.Rows, pred func(*storage.NetworkFlowPr
 				DstPort:    port,
 				L4Protocol: protocol,
 			},
-			LastSeenTimestamp: ts,
-			ClusterId:         clusterID,
+			ClusterId: clusterID,
```

Comment:
how about keeping more of old code and move the condition up

Reply:
It would leave a variable with gogo proto type.